Makefile.Common

```diff
@@ -1,3 +1,5 @@
+SHELL=/bin/bash -o pipefail
```

Comment:
Does it mean we route all `stdout` to `stderr`?

Reply:
Nah, so this just forces that bash keeps track of the entire pipe command instead of the last entry in the pipe command group.

Reply:
Can you please add a comment on what it does and why it's required for the test?

Reply:
Let me know if that comment makes sense.

Reply:
That's great. Thank you!